docs/superpowers/specs/2026-05-12-issue-85-shared-demod-dsp-modules-design.md

```
@@ -0,0 +1,229 @@
+  `ycbcr_to_rgb` `#[doc(hidden)]`. (#85; audit B1/B3/B5/B8/B16/C6/C20.)"*
+
+**Verification** — full local CI gate, with extra care given the wide blast radius:
+
+- `cargo test --all-features --locked --release` — the big one;
+  `tests/roundtrip.rs` exercises every mode's per-channel demod, so any
+  regression surfaces there.
+- `cargo fmt --all -- --check`
+- `cargo clippy --all-targets --all-features -- -D warnings`
+- `RUSTDOCFLAGS=-D warnings cargo doc --no-deps --all-features`
+- Sanity grep — after the refactor, this should print **nothing**:
+  ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)